cloudstack/resource_cloudstack_service_offering.go

```diff
@@ -42,6 +42,61 @@ func resourceCloudStackServiceOffering() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"cpu_number": {
+				Description: "Number of CPU cores",
+				Type:        schema.TypeInt,
+				Optional:    true,
+				ForceNew:    true,
+			},
+			"cpu_speed": {
+				Description: "Speed of CPU in Mhz",
+				Type:        schema.TypeInt,
+				Optional:    true,
+				ForceNew:    true,
+			},
+			"host_tags": {
+				Description: "The host tag for this service offering",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"limit_cpu_use": {
+				Description: "Restrict the CPU usage to committed service offering",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				ForceNew:    true,
+				Default:     false,
+			},
+			"memory": {
+				Description: "The total memory of the service offering in MB",
+				Type:        schema.TypeInt,
+				Optional:    true,
+				ForceNew:    true,
+			},
+			"offer_ha": {
+				Description: "The HA for the service offering",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				ForceNew:    true,
+				Default:     false,
+			},
+			"storage_type": {
+				Description: "The storage type of the service offering. Values are local and shared",
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Default:     "local",
```

Comment:
```suggestion
				Default:     "shared",
```
shared is the default for the API. Let's keep the same here.